pkg/controller/usersignupcleanup/usersignup_cleanup_controller_test.go

```diff
@@ -0,0 +1,172 @@
+		// accept any value here between the range of 89 days and 91 days
+		durLower := time.Duration(89 * time.Hour * 24)
+		durUpper := time.Duration(91 * time.Hour * 24)
+
+		require.Greater(t, res.RequeueAfter, durLower)
+		require.Less(t, res.RequeueAfter, durUpper)
+	})
+
+	t.Run("test that an old, deactivated UserSignup is deleted", func(t *testing.T) {
+
+		userSignup := test2.NewUserSignup(
+			test2.DeactivatedWithLastTransitionTime(threeYears),
+			test2.CreatedBefore(threeYears),
+			test2.WithStateLabel(v1alpha1.UserSignupStateLabelValueApproved),
+			test2.ApprovedAutomatically(),
+		)
+
+		r, req, _ := prepareReconcile(t, userSignup.Name, userSignup)
+
+		_, err := r.Reconcile(req)
+		require.NoError(t, err)
+
+		// Confirm the UserSignup has been deleted
+		key := test.NamespacedName(test.HostOperatorNs, userSignup.Name)
+		err = r.client.Get(context.Background(), key, userSignup)
+		require.Error(t, err)
+		require.True(t, errors.IsNotFound(err))
+		statusErr := err.(*errors.StatusError)
+		require.Equal(t, fmt.Sprintf("usersignups.toolchain.dev.openshift.com \"%s\" not found", key.Name), statusErr.Error())
+	})
+
+	t.Run("test that an old, unverified UserSignup is deleted", func(t *testing.T) {
+
+		userSignup := test2.NewUserSignup(
+			test2.CreatedBefore(threeYears),
+			test2.VerificationRequired(),
+		)
+
+		r, req, _ := prepareReconcile(t, userSignup.Name, userSignup)
+
+		_, err := r.Reconcile(req)
+		require.NoError(t, err)
+
+		// Confirm the UserSignup has been deleted
+		key := test.NamespacedName(test.HostOperatorNs, userSignup.Name)
+		err = r.client.Get(context.Background(), key, userSignup)
+		require.Error(t, err)
+		require.True(t, errors.IsNotFound(err))
+		require.IsType(t, &errors.StatusError{}, err)
+		statusErr := err.(*errors.StatusError)
+		require.Equal(t, fmt.Sprintf("usersignups.toolchain.dev.openshift.com \"%s\" not found", key.Name), statusErr.Error())
+	})
+
+	t.Run("test that an old, verified but unapproved UserSignup is not deleted", func(t *testing.T) {
+
+		userSignup := test2.NewUserSignup(
+			test2.CreatedBefore(threeYears),
+		)
+		userSignup.Spec.VerificationRequired = false
+		userSignup.Spec.Approved = false
+
+		r, req, _ := prepareReconcile(t, userSignup.Name, userSignup)
+
+		res, err := r.Reconcile(req)
+		require.NoError(t, err)
+
+		// Confirm the UserSignup has not been deleted
+		key := test.NamespacedName(test.HostOperatorNs, userSignup.Name)
+		err = r.client.Get(context.Background(), key, userSignup)
+		require.NoError(t, err)
+		require.NotNil(t, userSignup)
+		require.False(t, res.Requeue)
+	})
+
+}
+
+func prepareReconcile(t *testing.T, name string, initObjs ...runtime.Object) (*ReconcileUserSignupCleanup, reconcile.Request, *test.FakeClient) {
+	metrics.Reset()
+
+	s := scheme.Scheme
+	err := apis.AddToScheme(s)
+	require.NoError(t, err)
+
+	initObjs = append(initObjs)
+
```

Comment:
I'm not sure what is the purpose of this append, but it actually doesn't do anything, does it? 
```suggestion
```

Reply:
This was a copy/paste from another test.  I've removed it.